.babelrc.js

```diff
@@ -19,6 +19,7 @@ const config = {
 					browsers: [ 'last 2 versions', 'Safari >= 10', 'iOS >= 10', 'ie >= 11' ],
 				},
 				exclude: [ 'transform-classes', 'transform-template-literals' ], // transform-classes is added manually later.
+				useBuiltIns: 'entry',
```

Comment:
The [other option here](https://www.npmjs.com/package/@babel/preset-env#usebuiltins) is `usage`.

Which _might_ improve bundle size.

Reply:
usage won't help because Set and Map will only get polyfilled if they get used in _our_ code. Since we need the polyfill for code in node_modules that's not getting transpiled, it was be prone to breakage.

Reply:
IE it would work when some of our code that needs set or map runs before react does.

Reply:
I tried taking out the `useBuiltIns` line entirely and it still works to allow log in.  Should we take it out?

Reply:
nah, leave it in. Right now it won't make a huge diff, but down the road if and when we start doing different builds for new and old browsers, it will. 

babel-preset-env rewrites it based on the target browsers to only include the polyfills that are necessary for that set.